Cartfile

```diff
@@ -19,3 +19,5 @@ github "duckduckgo/swifter" == 1.4.3
 
 github "duckduckgo/lottie-ios" "tags/2.5.0"
 
+github "duckduckgo/PunycodeSwift" "brindy/app_extension_api_flag"
```

Comment:
This prevents a warning about linking to libraries that don't have the "Allow safe API extension" flag set. I've raised this with the project owner (and will probably create PR for it myself).  We can update this to a tag once that's done.